tests/credentials.test.ts

```diff
@@ -0,0 +1,120 @@
+import * as nock from "nock";
+import { Credentials, CredentialsMethod, DEFAULT_TOKEN_ENDPOINT_PATH } from "../credentials";
+import { AuthCredentialsConfig } from "../credentials/types";
+import { TelemetryConfiguration } from "../telemetry/configuration";
+import {
+  OPENFGA_API_AUDIENCE,
+  OPENFGA_CLIENT_ID,
+  OPENFGA_CLIENT_SECRET,
+} from "./helpers/default-config";
+
+nock.disableNetConnect();
+
+describe("Credentials", () => {
+  const mockTelemetryConfig: TelemetryConfiguration = new TelemetryConfiguration({});
+
+  describe("Refreshing access token", () => {
```

Comment:
These tests verify the token endpoint URL construction, but don't verify the JWT aud claim for PrivateKeyJWT flow. Consider adding tests that verify the audience matches the normalized issuer for different schemes (http://, https://, no scheme).